### ISSUE TITLE
compute resource nickname

### ISSUE THREAD

**Issue by bendichter** [scratchrealm/protocaas#7]
<img width="784" alt="image" src="https://github.com/scratchrealm/protocaas/assets/844306/55ffc6d1-3b53-468d-bb71-4aa2443e0c2f">

Representing each compute resource with a random string of characters does not feel that user friendly. This seems like the right approach for the backend, but would it be possible to get the user to name each compute resource with a nickname unique to the project and display that here?

This is true in general- I don't really like seeing these random strings of characters in the UI and I think in each case there is probably a more user-friendly way to display this information

**Comment by magland** [comment]
Yeah the compute resources, projects and workspaces all have names. I'll keep this open until this is fixed.

Jobs and batches on the other hand don't have names. We can think about that.

**Comment by magland** [comment]
@bendichter This has been fixed for compute resource display throughout. Let me know if you see any other instances of this, or have ideas on how to reduce the amount of random text.
